pkg/workflow/v1/validate_test.go

```diff
@@ -0,0 +1,33 @@
+package v1_test
+
+import (
+	"io/ioutil"
+	"testing"
+
+	v1 "github.com/puppetlabs/nebula-sdk/pkg/workflow/v1"
+	"github.com/stretchr/testify/require"
+)
+
+func TestValidYaml(t *testing.T) {
+	b, readErr := ioutil.ReadFile("testdata/valid.yaml")
```

Comment:
The Mac case-insensitive file system makes me feel angry inside.

Reply:
I'm sure trolling the nerdz was part of the design decision